### PR TITLE
Adds celsius to the freezer/heater, crafting update

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Freezers/heaters now also display the temperature in celsius (https://github.com/ParadiseSS13/Paradise/issues/332).
2) Updates our table crafting system. Also adds FRAG-12 shells and improvised shotguns.
3) Ensures CentCom telecomms doesn't use power.
4) Minor AI core fix, that might fix the latejoining issue.

### CODE REVIEW

**Comment by fj45** [comment]
Thanks mate!

**Comment by Artorp** [comment]
Nice! I was just looking into this too, and am noticing a couple things.

You need to add in ui data for the heater as well as the freezer, they use the same template.

Could change round(x) to round(x,1), otherwise you'll get -274 C which looks a bit silly.

Could also get rid of that one `&nbsp;` (causes small clipping at specific temps), I'll add a line note.

File: nano/templates/freezer.tmpl [hunk 1]
```diff
@@ -39,7 +39,7 @@
 			{{:helper.link('-', null, {'temp' : -100}, (data.targetGasTemperature > data.minGasTemperature) ? null : 'disabled')}}
 			{{:helper.link('-', null, {'temp' : -10}, (data.targetGasTemperature > data.minGasTemperature) ? null : 'disabled')}}
 			{{:helper.link('-', null, {'temp' : -1}, (data.targetGasTemperature > data.minGasTemperature) ? null : 'disabled')}}				
-			<div style="float: left; width: 80px; text-align: center;">&nbsp;{{:data.targetGasTemperature}} K&nbsp;</div>
+			<div style="float: left; width: 110px; text-align: center;">&nbsp;{{:data.targetGasTemperature}} K&nbsp; ({{:data.targetGasTemperatureCelsius}} &deg;C)</div>
```
Comment (Artorp) [inline nano/templates/freezer.tmpl]: The `&nbsp;` between `center;">` and `{{:data` doesn't really do anything and shifts the text center.

Comment (Artorp) [inline nano/templates/freezer.tmpl]: Actually both `&nbsp;`s could be removed, everything's wrapped up in a div with fixed width so line breaks shouldn't be a concern.

**Comment by Markolie** [comment]
Ah, you're right, I overlooked the heater. That's silly of me. 

I'm not sure how much the &nbsp; matters, as it's just a regular space. I'll replace it with a space though, no huge different.

As for the round, I don't think that'll solve the issue of it displaying -274C. I'll have to add a check to see if the total_moles (and just to make sure, the temperature) are 0, and if so, set the Celsius temperature to 0.

I probably shouldn't be coding this many things at once :P.

**Comment by Artorp** [comment]
Yeah that space was shifting the whole text a bit, caused clipping at max characters (eg. 173 K, -100 C). Not an issue by any means, just thought it was worth a mention.

I don't mind it showing a temperature when there's no air, it's just that round(x) behaves like floor(x) unless you give it a second argument, which means -273.15 will display as -274 C (impossible), instead of -273 C. Wouldn't mind a check for no gas, but I don't think that's necessary (in that case, passing something like n/a would make more sense than 0).

Oh and could you do an overhaul of the whole atmos system while you're at it? :P

**Comment by Markolie** [comment]
Well, if there's no gas, it currently also displays 0 K, so I'll just stick with that. And very well, I'll add the round parameter.

And one day I do hope to port LINDA. One day...

File: code/game/objects/items/weapons/grenades/ghettobomb.dm [hunk 2]
```diff
@@ -82,14 +55,10 @@
 				prime()
 
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
-//	update_mob()
-//	explosion(epicenter, 1, 2, 3, 3)
-	explosion(src.loc,0,0,2,2)
-	del(src)
+	update_mob()
+	explosion(src.loc,-1,-1,-1, flame_range = range)	// no explosive damage, only a large fireball.
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/grenades/ghettobomb.dm]: You'll want to use our old explosive values here; we don't have flame_range (at least not until we get LINDA), so this bomb will literally do nothing, hahah.

**Comment by Fox-McCloud** [comment]
Quite frankly, I'm ok with this being merged in earlier rather than later, as table construction is currently prone to runtiming and there's very little in terms of non-QoL (celsius) features here.

Only thing that's really being added is the shotgun and frag shells (and the improvised shotgun item has been in for over a month now)

**Pull request merged**